### PR TITLE
Added details of rich text field.

### LINKED ISSUE #4294: Umbraco Forms v10 zip does not include FieldType.RichText.cshtml
The Umbraco Forms v10 theme download zip file does not include the new rich text field type view "FieldType.RichText.cshtml".

***This is the page with issues: https://github.com/umbraco/UmbracoDocs/edit/main/Add-ons/UmbracoForms/Developer/Themes/index.md***

Details also not included in field types overview page - https://our.umbraco.com/documentation/Add-ons/umbracoforms/editor/creating-a-form/fieldtypes/

### PR DESCRIPTION
**Pull request by AndyButland**
This fixes #4294 by adding details of the rich text field to the page that lists all the fields.

Note that I've also already fixed the link so we reference the latest version of the files (sorry, I did that by accident direct in `main`).

### CODE REVIEW

File: Add-ons/UmbracoForms/Editor/Creating-a-Form/Fieldtypes/index.md [hunk 1]
```diff
@@ -61,12 +61,17 @@ By default, the following Field Types are available:
 - **Single Choice**:
     Displays a list of items with a radio button for each item where the user can select a single option.
 
-    ![singlechoice](images/singlechoice-v9.png)
+    ![Single choice](images/singlechoice-v9.png)
 
 - **Title and Description**:
-    Displays a title and description that are set as default values.
+    Displays a read-only title and description for a set of form fields.
```
Comment (eshanrnh) [inline Add-ons/UmbracoForms/Editor/Creating-a-Form/Fieldtypes/index.md:67]:
```suggestion
    Displays a read-only title and description for a set of Form fields.
```

File: Add-ons/UmbracoForms/Editor/Creating-a-Form/Fieldtypes/index.md [hunk 2]
```diff
@@ -61,12 +61,17 @@ By default, the following Field Types are available:
...
 
-    ![Radiobuttonlist](images/titleanddescription-v9.png)
+    ![Title and description](images/titleanddescription-v9.png)
+
+- **Rich Text**:
+    Displays read-only formatted text that can be used to provide additional copy within a form.
```
Comment (eshanrnh) [inline Add-ons/UmbracoForms/Editor/Creating-a-Form/Fieldtypes/index.md:72]:
```suggestion
    Displays a read-only formatted text that can be used to provide an additional copy within a Form.
```

File: Add-ons/UmbracoForms/Editor/Creating-a-Form/Fieldtypes/index.md (hunk 1 above)
Comment (eshanrnh) [inline Add-ons/UmbracoForms/Editor/Creating-a-Form/Fieldtypes/index.md:67]:
```suggestion
    Displays a read-only title and description for a set of Form fields.
```

**Review by eshanrnh — APPROVED** [review]
Looks good to me, @AndyButland 🙌 Just minor suggestions.

File: Add-ons/UmbracoForms/Editor/Creating-a-Form/Fieldtypes/index.md (hunk 1 above)
Comment (AndyButland) [reply Add-ons/UmbracoForms/Editor/Creating-a-Form/Fieldtypes/index.md:67]: I'd be minded to keep this as "form" as here we are talking about "a form" rather than "Umbraco Forms" the product.  But up to you.

File: Add-ons/UmbracoForms/Editor/Creating-a-Form/Fieldtypes/index.md (hunk 2 above)
Comment (AndyButland) [reply Add-ons/UmbracoForms/Editor/Creating-a-Form/Fieldtypes/index.md:72]: As above.  Also, here I don't think we want the "an".  I'm looking to say here you can add additional copy in the sense of "additional formatted text" rather than a copy as in "a duplicate".

Comment (eshanrnh) [reply Add-ons/UmbracoForms/Editor/Creating-a-Form/Fieldtypes/index.md:72]: Oh ok, now it makes sense to me 😃I was thinking of it as an additional duplicate copy. 

Can we maybe rephrase it to make it more clear? Maybe something like: 
```suggestion
    Displays a read-only tool that can be used to provide additional formatted text within a Form.
```

File: Add-ons/UmbracoForms/Editor/Creating-a-Form/Fieldtypes/index.md (hunk 1 above)
Comment (eshanrnh) [reply Add-ons/UmbracoForms/Editor/Creating-a-Form/Fieldtypes/index.md:67]: Makes sense.

File: Add-ons/UmbracoForms/Editor/Creating-a-Form/Fieldtypes/index.md (hunk 2 above)
Comment (AndyButland) [inline Add-ons/UmbracoForms/Editor/Creating-a-Form/Fieldtypes/index.md:72]: Not sure "tool" is right... how about:

```suggestion
    Displays read-only formatted text that can be used to provide additional information and links within a form.
```
